### DISCUSSION TITLE
Seeing gevent-websocket error in logs but web server works

### DISCUSSION THREAD

**Discussion by rajan-code** [miguelgrinberg/Flask-SocketIO#2007] [General]
I'm using flask-socketio and gevent   
At the top:
```
from gevent import monkey 
monkey.patch_all()
```

`socketio = SocketIO(app, cors_allowed_origins=['https://example.com', "http://localhost:3000"], async_mode='gevent')`
I pip installed the latest versions of gevent and gevent-websocket
I followed the deployment instructions in [the docs](https://flask-socketio.readthedocs.io/en/latest/) and am using uwsgi:
`uwsgi --http :8000 --gevent 500 --processes 1 --logto /tmp/backend.log --stats /tmp/statsock --http-websockets --master --wsgi-file app.py --callable app`

I see this error message in /tmp/backend.log:
```raise RuntimeError('You need to use the gevent-websocket server. '
RuntimeError: You need to use the gevent-websocket server. See the Deployment section of the documentation for more information.```
My web server seems to be working fine though. Should I just ignore this message?

**Comment by miguelgrinberg** [comment]
async_mode should be `gevent_uwsgi`.

**Reply by rajan-code** [reply]
Thanks  
Btw is the --gevent option the max number of active greenlets?

**Reply by miguelgrinberg** [reply]
Yes. You should probably read the documentation if this is something you aren't familiar with: https://uwsgi-docs.readthedocs.io/en/latest/Options.html#gevent
